src/Stratis.Bitcoin.Features.Wallet/Wallet.cs

```diff
@@ -728,12 +728,29 @@ public IEnumerable<UnspentOutputReference> GetSpendableTransactions(int currentC
 
                     if (confirmationCount >= confirmations)
                     {
```

Comment:
I would flip the condition here to just `continue` so that the nested `if` can be avoided.